client/src/components/Users.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+
+const UserList = styled.div`
+  position: relative;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+`;
+
+class Users extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      users: [],
+      errorMessage: '',
+    };
+  }
+
+  componentDidMount() {
+    let url;
+    if (this.props.department) {
+      url = 'http://localhost:4200/users/department'
+    } else {
+      url = 'http://localhost:4200/users';
+    }
+
+    axios.defaults.withCredentials = true;
+    axios
+      .get(`${url}`, {
```

Comment:
Pretty sure you could change this to just `.get(url, {` since it's already a string.